ucm2/alcatel-idol3/HiFi.conf

```diff
@@ -0,0 +1,58 @@
+# Use case configuration for Alcatel OneTouch Idol 3.
+# The two speakers/earpieces are connected to external codec (TFA9897)
+# Headphones support requires writing a driver for AK4375
+
+Define {
+	WcdPlaybackPCM "hw:${CardId},1"
```

Comment:
try removing this, it shouldn't be needed since you don't use pm8916 for playback